task2.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+    "fmt"
+    "sort"
+)
+
+type Point struct {
+  x, y int
+}
+
+type Square struct {
+  start Point
+  a uint
+}
+
+func (s Square) End() Point {
+  return Point{x: s.start.x + int(s.a), y: s.start.y - int(s.a)}
+}
+
+func (s Square) Perimeter() uint{
+  return 4 * s.a
+}
+
+func (s Square) Area() uint {
+  return s.a * s.a
+}
+
+func median(i []int) float64 {
+  sort.Ints(i)
+  if len(i) % 2 == 0 {
+    return (float64(i[len(i) / 2 - 1]) + float64(i[len(i) / 2])) / 2
```

Comment:
You have a `return` here, so you don't need `else` statement. Please make it look something like:
```
if len(i) % 2 == 0 {
        return ...
}

return ...
```